frontend/src/components/pages/login/Login.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React  from 'react';
+import Login from './Login';
+import { render, fireEvent, waitForElement } from "@testing-library/react";
+
+// const { createBrowserHistory } =  require('history');
+// const history = createBrowserHistory(/* ... */);
+// jest.spyOn(history, "push");
```

Comment:
you can use jest.mock('../../../services/sessionStorageService', () => { getUserFromStorage: () => 'mock' });
Where mock is a value you want to have in test